apps/api/src/app/events/usecases/send-message/send-message-email.usecase.ts

```diff
@@ -209,6 +209,21 @@ export class SendMessageEmail extends SendMessageType {
       })
     );
 
+    if (!html) {
+      await this.createExecutionDetails.execute(
```

Comment:
Combining with my comment from above regarding catching in the actual send message usecase and not compile template. We need to add this execution detail to all channels, so maybe consider moving this catching to one level above?